js/customize-snapshots.js

```diff
@@ -1113,6 +1113,7 @@
 				isSameStatus = api.state( 'changesetStatus' ).get() === originalOptions.data.customize_changeset_status;
 				if ( 'customize_save' === originalOptions.data.action && isSameStatus && options.data ) {
 					options.data = removeParam( options.data, 'customize_changeset_status' );
```

Comment:
As we are checking `customize_snapshots_create_revision` `customize_changeset_status ` does not need to be removed now.